package.json

```diff
@@ -29,11 +29,8 @@
   "dependencies": {
     "classnames": "^2.1.2",
     "object-assign": "^4.0.1",
-    "soundcloud-audio": "^1.0.3"
-  },
-  "peerDependencies": {
-    "react": "^0.14.x",
-    "react-dom": "^0.14.x"
+    "soundcloud-audio": "^1.0.3",
+    "react": "^0.14.x"
```

Comment:
@joewestcott  why `react` and `soundcloud-audio` to peerDependencies? I was thinking to put everything into deps, what do you think?